configs/pulpcore/3.18.yaml

```diff
@@ -0,0 +1,78 @@
+---
+:project: pulpcore
+:github_org: pulp
+:nightly: false
+:repos: []
+:releases:
+  :3.18:
+    :redmine_version_id: -1
+:mash_scripts:
+  - pulpcore-3.18-el8
+:strict_keys: true
+:gpg_key: C9308E1F
+:tags:
+  - name: pulpcore-3.18-el8
+    based_off: pulpcore-3.17-el8
+    helper_tags:
+      pulpcore-3.18-el8-override: null
+    build_target: pulpcore-3.18-el8-build
+    build_package_group_source_tag: pulpcore-3.18-el8-build
+    arches:
+      - x86_64
+    inherits:
+      pulpcore-3.18-el8-build:
+        pulpcore-3.18-el8-override: 0
```

Comment:
Any reason not to inherit from pulpcore-3.17?

Reply:
Looking at the configs for 3.17 and 3.16, it does inherit the Current version not the previous one. This is probably the inheritance to compose the packages.